tests/modules/s3_datasets_shares/conftest.py

```diff
@@ -206,6 +211,29 @@ def factory(dataset: S3Dataset, name, username) -> DatasetTable:
     yield factory
 
 
+@pytest.fixture(scope='module')
+def table_data_filter(db):
+    def factory(
+        table: DatasetTable,
+        name,
+        filterType,
+    ) -> DatasetTableDataFilter:
+        with db.scoped_session() as session:
+            data_filter = DatasetTableDataFilter(
+                tableUri=table.tableUri,
+                label=name,
+                filterType=filterType,
+                rowExpression=None,
+                includedCols=['id1', 'id2'],
+                owner='foo',
+            )
```

Comment:
This can only be a COLUMN filter, as it is tested in the tests. Maybe we need make it more explicit to avoid unexpected errors down the line

Reply:
updated naming to reflect col data filter